modules/client/src/client-common/utils.ts

```diff
@@ -48,7 +48,10 @@ export function computeProposalStatus(
   if (startDate >= now) {
     return ProposalStatus.PENDING;
   }
-  if (proposal.potentiallyExecutable || proposal.earlyExecutable) {
+  if (
+    (proposal.potentiallyExecutable || proposal.earlyExecutable) &&
+    endDate >= now
+  ) {
     return ProposalStatus.SUCCEEDED;
   }
   if (endDate >= now) {
```

Comment:
But now, this means that a passed proposal that was not executed will be considered as DEFEATED, which is not correct. Why would an executable proposal be displayed as defeated?

If the plugin won't allow it to be executed after expired, then we would need another state

Reply:
I dont think so, if is potentially executable and the date expired it will appear as succeeded. Maybe I should give it a go with the tests ant test more edge cases but i think tthe change is ok

Reply:
Well, the execution of an executable proposal that already ended would not enter any of the "if" clauses. It would keep trying until the end of the function, when it would simply return DEFEATED